### PR TITLE
Servlets

### PR DESCRIPTION
**Pull request by sammurraytuesta**
Implement a servlet to display a message onto page as well as a JSON application to display a random hardcoded message.

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
```
Comment (rjkroege) [inline portfolio/pom.xml]: I don't understand why the jetty version is lower.

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 2]
```diff
@@ -5,14 +5,25 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import com.google.gson.Gson;
```
Comment (rjkroege) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:8]: Please sort per https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java [hunk 3]
```diff
@@ -5,14 +5,25 @@
...
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+
+    ArrayList<String> AList = new ArrayList<String>();
+    AList.add("Hello World!");
+    AList.add("How are you?");
+    AList.add("How's it going?");
+
+    Gson gson = new Gson();
```
Comment (rjkroege) [inline portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:24]: Excellent choice to use the JSON library.

File: portfolio/src/main/webapp/servlet.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+/** Fetches the current date from the server and adds it to the page. */ 
+
```
Comment (rjkroege) [inline portfolio/src/main/webapp/servlet.js]: Remove the blank line per https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -7,3 +7,9 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+#hello-container{  
+    font-size: 2vh;
```
Comment (rjkroege) [inline portfolio/src/main/webapp/style.css]: `vh` for font size was new to me. This seems clever. (No action needed.)

File: walkthroughs/week-1-web-development/examples/stanley/pom.xml [hunk 6]
```diff
@@ -12,7 +12,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <jetty.version>11.0.0</jetty.version>
+    <jetty.version>9.4.31.v20200723</jetty.version>
```
Comment (rjkroege) [inline walkthroughs/week-1-web-development/examples/stanley/pom.xml]: See my other comment: why is the jetty version downgraded?

**Review by rjkroege — CHANGES_REQUESTED** [review]

File: portfolio/pom.xml (hunk 1 above)
Comment (sammurraytuesta) [reply portfolio/pom.xml]: Yes haha this was due to an implementation error that was communicated to us by kevin! Thank you!

File: walkthroughs/week-1-web-development/examples/stanley/pom.xml (hunk 6 above)
Comment (sammurraytuesta) [reply walkthroughs/week-1-web-development/examples/stanley/pom.xml]: Resolved in other comment :)

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/style.css]: Thank you! I find it to be easier to adjust to screen size :)

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java (hunk 3 above)
Comment (sammurraytuesta) [reply portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:24]: Thank you!

File: portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java (hunk 2 above)
Comment (sammurraytuesta) [reply portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java:8]: I think I reordered them correctly but if not let me know! Thank you!

File: portfolio/src/main/webapp/servlet.js (hunk 4 above)
Comment (sammurraytuesta) [reply portfolio/src/main/webapp/servlet.js]: Removed! Thank you!

**Comment by sammurraytuesta** [comment]
I will resolve conflicts after the design PR is merged!

**Pull request merged**